docs/source/onboarding_resources/tutorial/index.ipynb

```diff
@@ -0,0 +1,1653 @@
+{
```

Comment:
<p>In  response to the second paragraph of the risk factor: In the bullets at the top about the UC Davis data you state "31% lower prevalence of severe wasting" which would kinda undercut this line. I don't think it's wrong to make this assumption still, but we do seem to have some data supporting the concept of greater effects in more severe cases.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research/pull/1579/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Reply:
<p>This is an excellent point and I believe suggests a simple improvement to the SQ-LNS model! Thanks for pointing it out.</p>

Reply:
Ok, I decided this is actually just a bit too hard and I don't want to add the complexity to the model. I'm going to add a note instead.